runtime/android/core_internal/src/org/xwalk/core/internal/ReflectionHelper.java

```diff
@@ -2,16 +2,53 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+/*
```

Comment:
Is this piece of code dual-licensed ASF and the BSD-style license in the LICENSE file, or was this license addition done in error?